docs/Plugins/jenkins.md

```diff
@@ -12,6 +12,9 @@ This plugin collects Jenkins data through [Remote Access API](https://www.jenkin
 - What is the build success rate?
 - How long does it take for a code change to be deployed into production?
 
+## Supported Versions
+Versions 2.263.x+
```

Comment:
Available for Jenkins v2.263.x+. Check [this doc](relative url of the supported data sources) for more details.